### DISCUSSION TITLE
Query About Missing Repository: NangateOpenCellLibrary

### DISCUSSION THREAD

**Discussion by Vishnu460** [The-OpenROAD-Project/OpenROAD#6398] [General]
Hello OpenROAD Team,

I am trying to access the repository for the NangateOpenCellLibrary, but the URL (https://github.com/The-OpenROAD-Project/NangateOpenCellLibrary) seems to return a 404 error. Could you please let me know if this repository has been moved or renamed?

Thank you for your help!

Best regards,  
vishnu tammineni
9154628433

**Comment by maliberty** [comment]
There is no such library, why do you think there is one?

**Reply by Vishnu460** [reply]
I am currently working with OpenROAD and attempting to integrate the Nangate45 PDK for my project. I noticed that the repository URL for the NangateOpenCellLibrary (https://github.com/The-OpenROAD-Project/NangateOpenCellLibrary) leads to a 404 error.

Could you please clarify if:

This repository has been moved to a different location?
There is an alternative way to access the NangateOpenCellLibrary?
Any recommended steps to acquire the Nangate45 library for use with OpenROAD?
Your guidance would be greatly appreciated.

Thank you for your support!

Best regards,

**Reply by maliberty** [reply]
There was never such a repository to my knowledge.  Nangate45 is also part of ORFS https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/platforms/nangate45
